### ISSUE TITLE
LocalCluster in processes=False doesnt accept  the keyword worker_class

### ISSUE THREAD

**Issue by redsum** [dask/distributed#2447]
this will fail

`cluster = LocalCluster(processes=False,  memory_limit=None, worker_class=Worker)`

while this will pass

`cluster = LocalCluster(processes=True,  memory_limit=None, worker_class=Worker)`

**Comment by mrocklin** [comment]
Yes, the `processes=` keyword is really just syntactic sugar around using the `Nanny` rather than the `Worker` class.  

What is it that you're trying to achieve?

**Comment by redsum** [comment]
I'm using multiprocessing, and i want to have a custom initialization for each process 
So i use the c'tor of the worker to perform the initialization

But the main point here is that There shouldn't be a difference in API of multiprocessing and Multithreading
